stm32-modules/include/heater-shaker/heater-shaker/heater_task.hpp

```diff
@@ -402,24 +406,28 @@ class HeaterTask {
         }
         // NOLINTNEXTLINE(cppcoreguidelines-avoid-magic-numbers)
         if (state.system_status == State::CONTROLLING) {
-            if (!policy.set_power_output(
-                    pid.compute(setpoint.value() - pad_temperature()))) {
+            uint8_t error = policy.set_power_output(pid.compute(setpoint.value() - pad_temperature()));
+            if (error) {
                 state.system_status = State::ERROR;
                 setpoint = std::nullopt;
-                state.error_bitmap |= State::CIRCUIT_ERROR;
-                auto error_message =
-                    messages::HostCommsMessage(messages::ErrorMessage{
-                        .code =
-                            errors::ErrorCode::HEATER_HARDWARE_ERROR_CIRCUIT});
+                auto error_message = messages::ErrorMessage{};
+                if (error == 9){
```

Comment:
let's get some names for these values instead of just integer constants please. ideally they could be in a header shared by both c and c++.